website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "March IDE updates and fixes"
+id: "mar-ide-updates"
+description: "Mar 2023 release note: We've enhanced the IDE by adding add common dbt commands to the command palette, creating PRs even if you have uncommitted changes, autocompleting suggestions when editing a yml file, editing directly in the git diff view, improved the DAG selector, upgraded sqlfmt, improved syntax error messaging, and more."
+sidebar_label: "Update and fixes: IDE"
+tags: [Mar-31-2023, IDE]
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continue to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+
+## New features 
+
+- You can now commit and revert individual files under **Version Control**.
```

Comment:
Just a small suggestion - The assumption of the new features is that it's something they can do now which previously could not be done.
```suggestion
- Commit and revert individual files under **Version Control**.
```